aspnetcore/web-api/advanced/analyzers.md

```diff
@@ -12,6 +12,20 @@ uid: web-api/advanced/analyzers
 
 ASP.NET Core 2.2 and later includes the [Microsoft.AspNetCore.Mvc.Api.Analyzers](https://www.nuget.org/packages/Microsoft.AspNetCore.Mvc.Api.Analyzers) NuGet package containing analyzers for web APIs. The analyzers work with controllers annotated with <xref:Microsoft.AspNetCore.Mvc.ApiControllerAttribute>, while building on [API conventions](xref:web-api/advanced/conventions).
```

Comment:
@pranavkm Should this paragraph's first sentence change too? I assume the analyzer package shouldn't be referenced in ASP.NET Core 2.2 projects if you're using the .NET Core 3.0 SDK or later.

Reply:
I guess. Do you have suggestions on how to phrase it - we still want a short introduction about what the analyzers do and how they light up.

Reply:
bump

Reply:
Apologies for the delay. I'll make an attempt at explaining it in this PR and will ping you for review when ready.